dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/service/DefaultLinkService.java

```diff
@@ -69,18 +70,9 @@ public class DefaultLinkService implements LinkService
 
     private final ContextService contextService;
 
-    public DefaultLinkService( SchemaService schemaService, ContextService contextService )
-    {
-        checkNotNull( schemaService );
```

Comment:
Should the dependencies then get a `@NonNull` annotation if you remove the check? as in https://github.com/dhis2/wow-backend/blob/master/guides/spring_conventions.md#injection

Reply:
We removed these checks in beans when switching to lombok annotations. Use of `@NonNull` (or any equivalent annotation) is a general question we might want to discuss in Thursday meeting. I think we could benefit using it where we feel it helps but we should agree which of the available annotations we start using.